docs/course.md

```diff
@@ -217,3 +213,24 @@ The default timezone for courses is `America/Chicago` (U.S. Central Time). This
 ```
 
 Allowable timezones are those in the TZ column in the [list of tz database time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), which is a display version of the [IANA Time Zone Database](https://www.iana.org/time-zones).
+
+## Comment
+
+You can add comments to json files using the property `comment`. You can use this property more than once in the same file. For example:
+
+```json
+{
+    "comment": "assessments that are shared among a group of students",
+    "assessmentSets": [
+      {"abbreviation": "GA", "name": "Group Activity", "heading": "Group Activities", "color": "green1"}
+    ],
+    "comment": "timezone for the course",
```

Comment:
We should delete this the second comment here. Although PL currently allows more than one comment per object, it actually silently drops all the comments except the last one if the file is automatically edited.

I don't currently have a good way to catch JSON files with repeated keys (like more than one comment in the same object). `JSON.parse()` silently drops all earlier copies of a key. As does `jju` that we use.

Reply:
Applied.